apps/examples/wc-react17/src/index.js

```diff
@@ -1,6 +1,5 @@
 import React from 'react';
-import ReactDOM from 'react-dom/client';
+import { render } from 'react-dom';
```

Comment:
Note to self - This had to be changed for some reason as `react-dom/client` couldn't be resolved. Not sure why as we haven't changed that dep, but will look into it.

The build passes in CI, but nothing is rendered, and there are a few errors in console.

Reply:
All working :)